.github/workflows/libfuzzer.yaml

```diff
@@ -0,0 +1,170 @@
+name: Libfuzzer
```

Comment:
Could you give the workflow a more descriptive name (e.g., Run Libfuzzer on PostgreSQL 15)?

Reply:
We usually keep these names short like "Regression", I guess to fit into the Github user interface. I'll add the postgres version to the job name, like in other tests.